jmix-superset/superset-starter/src/main/java/io/jmix/autoconfigure/superset/SupersetAutoConfiguration.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2024 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.autoconfigure.superset;
+
+import io.jmix.superset.SupersetConfiguration;
+import io.jmix.superset.SupersetProperties;
+import io.jmix.superset.SupersetTokenManager;
+import io.jmix.superset.client.cookie.SupersetCookieManager;
+import io.jmix.superset.schedule.SupersetTokenScheduleConfigurer;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
+import org.springframework.scheduling.TaskScheduler;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
+
+@AutoConfiguration
+@Import(SupersetConfiguration.class)
+public class SupersetAutoConfiguration {
+
+    @Bean("sprset_ThreadPoolCsrfTokenTaskScheduler")
+    @ConditionalOnProperty(value = "jmix.superset.csrf-protection-enabled", matchIfMissing = true)
```

Comment:
Shouldn't we also analyze the `jmix.superset.tokens-refresh-enabled` property value here? I mean that even if CSRF protection is enabled, but tokens refresh are disabled,  then this TaskScheduler won't be used by anyone?

Reply:
Fixed